client/src/components/MultiJurisdictionAudit/RoundManagement/index.tsx

```diff
@@ -56,22 +61,28 @@ const RoundManagement = ({ round, auditBoards, createAuditBoards }: IProps) => {
     auditBoards,
   ])
 
-  // const [{ online }] = useAuditSettings(electionId!)
+  const { online } = useAuditSettingsJurisdictionAdmin(
+    electionId,
+    jurisdictionId
+  )
 
-  if (!ballots) return null // Still loading
+  if (!ballots || online === null) return null // Still loading
 
   if (!round.isAuditComplete) {
     const { roundNum } = round
     return (
-      <Wrapper className="single-page left">
-        <H2Title>Round {roundNum} Audit Board Setup</H2Title>
-        <CreateAuditBoards
-          auditBoards={auditBoards}
-          createAuditBoards={createAuditBoards}
-          numBallots={ballots.length}
-          roundNum={roundNum}
-        />
-        {auditBoards.length > 0 && (
+      <PaddedWrapper className="single-page left">
+        {auditBoards.length === 0 ? (
```

Comment:
what's this change for?

Reply:
@jonahkagan The `PaddedWrapper` is created above to add a bottom margin: https://github.com/votingworks/arlo/pull/665/files/3e594ad5164b6998640979359b547715cdf67ac8#diff-4684ee384072240a8164a08c77c1e3c7R23

Reply:
oops, should have been more specific, I meant L75 - checking `auditBoards.length`